apps/settings/elements/accessibility_monoaudio.html

```diff
@@ -0,0 +1,19 @@
+<element name="accessibility-monoaudio" extends="section">
+  <template>
+
+    <gaia-header action="back" data-href="#accessibility">
```

Comment:
Back button needs to be localized see: https://github.com/gaia-components/gaia-header#localization